src/Graphics/Vty/Attributes/Color240.hs

```diff
@@ -1,264 +1,260 @@
 -- This header file was generated by ./256colres.pl
 module Graphics.Vty.Attributes.Color240
-  ( rgbColor
+  ( make240ColorCodeFromRGB
   , color240CodeToRGB
   )
 where
 
-import Graphics.Vty.Attributes.Color
-
 import Data.Word (Word8)
 import Text.Printf
 
 -- Note: rgbColor's mapping from RGB to 240 colors was generated from
 -- 256colres.pl which is forked from xterm 256colres.pl.
 
--- | Create a Vty 'Color' (in the 240 color set) from an RGB triple.
--- This function is lossy in the sense that we only internally support 240 colors but the
--- #RRGGBB format supports 16^3 colors.
-rgbColor :: Integral i => i -> i -> i -> Color
-rgbColor r g b
+-- | Create a value in the Color240 set from an RGB triple
+make240ColorCodeFromRGB :: Integral i => i -> i -> i -> Word8
+make240ColorCodeFromRGB r g b
```

Comment:
In the future, please make API changes in separate commits from changes like documentation, and clearly mark them in the commit message. That helps me make sure that when I write the release changelog from the commit history, I can more easily find breaking changes like this.

Also, please rename this to `rgbColorToColor240`.